### ISSUE TITLE
--help is broken when run via function

### ISSUE THREAD

**Issue by atombaby** [FredHutch/swift-switch-account#10]
```
mrg@rhino-d[~]: sw2account --help
-bash: shell: No such file or directory
```

Likely something with argument processing: the function calls sw2account.py:

```
    eval `/usr/sbin/sw2account.py bash $* ;`
```

**Comment by atombaby** [comment]
sw2account.py is properly rendering the help message.  However, that output is on STDOUT, which is passed to eval, which barfs on the help message.

**Comment by atombaby** [comment]
Not sure if this is the best way to correct this issue, but does seem to fix while requiring little of the shell function (which is kind of desirable):

override the "print_help" function in ArgumentParser object to send output to stderr.  Then eval won't see that output, so no error or action.  Error message is displayed onto the current stderr file handle.

fixed in commit 6436ad79449a867db2dc407e3879b1a7be8a10d9
